spec/integration/inventory2_spec.rb

```diff
@@ -453,27 +453,34 @@ def fact_plan(name = 'facts_test')
 
   # TODO: these tests require docker so they may as well require ssh for now
   context 'with pdb lookups', ssh: true, puppetdb: true do
+    let(:shell_cmd) { 'whoami' }
+    # This is an actual user - password is also 'test'
+    let(:user) { 'test' }
```

Comment:
Where is this user created/managed?

Reply:
https://github.com/puppetlabs/bolt/blob/master/spec/Dockerfile#L20

Reply:
we should probably add it to the helper libraries then